components/SideBar.jsx

```diff
@@ -160,6 +160,11 @@ export default function PersistentDrawerLeft () {
   const [openLisNotification, setOpenLisNotification] = useState(false)
   const selectedPage = useSelector(selectActiveSideBar)
 
+  useEffect(() => {
+    if (user.role !== 'admin') router.push('/login')
```

Comment:
a nghĩ chức năng này cần được viết ở master layout( 1 cái layout chung cho toàn side)
thứ 2 là cần check nó login vì check như này mỗi lần f5 chưa có reducer nó sẽ đá về login